metpy/plots/skewt.py

```diff
@@ -7,6 +7,7 @@
 `SkewT`, as well as a class for making a `Hodograph`.
 """
 
+import matplotlib as mpl
```

Comment:
How about `import matplotlib.colors as mcolors`? That would be more standard with how I usually do it.